accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,298 @@
+
+  Every update to the load balancer `nginx` will be handled via SUSE Manager.
+
+  ##### Bootstrapping an empty cluster
+
+  Salt will invoke `skuba` (using `cmd.run`) in the Management Node. Unfortunately, `skuba` does not offer an API yet (is in the progress).
+  Example:
+
+  ```
+  bootstrap_master:
+    cmd.run:
+      - name: skuba cluster init --control-plane <load balancer IP> <cluster name>
+  ```
+
+  The parameters above are already known to SUSE Manager.
+
+  It is probably not in the scope of this RFC to offer to the user the ability to customize the configuration directly from SUSE Manager, but it will be in the future. At this point, the configuration of the cluster is in the Management Node and can be customized by the user (e.g. [integrating an external LDAP](https://susedoc.github.io/doc-caasp/beta/caasp-deployment/single-html/#_integrate_external_ldap) or [configure `kured`](https://susedoc.github.io/doc-caasp/beta/caasp-deployment/single-html/#_prevent_nodes_running_special_workloads_from_being_rebooted)). 
+  
+  Single Sign-On is out of scope until further developments from the Product Management.
+  
+  ##### Master and worker nodes
+
+  ###### Registering master and worker to SUSE Manager
+
+  The core of the CaaS Platform is master and worker nodes. The Management Node, instructed by SUSE Manager, will take care of bootstrapping master and worker nodes.
+
+  The critical part of every master and worker node is handling updates of the underlying Kubernetes-related packages (`kubernetes-kubeadm kubernetes-kubelet kubernetes-client cri-o cni-plugins`): these updates must be handled with `skuba` from the Management Node.
+
+  [`skuba-update` will take care of updating the Base Operating System](https://susedoc.github.io/doc-caasp/beta/caasp-admin/single-html/#_base_os): this means we cannot register any master or worker node with SUSE Manager (otherwise an end-user could apply updates to the machine).
+  In case the machines are registered to SUSE Manager, the machines will be de-registered during the bootstrapping.
+  
+  ###### Bootstrapping master and worker nodes
+
+  In SUSE Manager under "CaaS Platform > Deployment > cluster name" there will be a list of machines that:
+  - are not part of any other CaaS Platform cluster
+  - are not already bootstrapped into the current cluster
+  - satisfy the CaaS Platform node requirements: those machines are either bootstrapped via the AutoYaST profile or are already registered with SUSE Manager and satisfy the node requirements (checked via grains).
+
+  The user will then have the option to bootstrap the selected machine as the first master (or subsequently, as an additional master or worker). Upon triggering of this event, the SUSE Manager will:
```

Comment:
I'd suggest to rename `master` to `control plane` nodes everywhere, it's the terminology being used in Kubernetes nowadays. It's true that `master` appears in many places though.

Reply:
Ok, renamed everything to `control plane`